src/tools/fmod_editor_export_plugin.h

```diff
@@ -0,0 +1,28 @@
+#ifdef TOOLS_ENABLED
+
+#ifndef GODOTFMOD_FMOD_EDITOR_EXPORT_PLUGIN_H
+#define GODOTFMOD_FMOD_EDITOR_EXPORT_PLUGIN_H
+
+#include <classes/editor_export_plugin.hpp>
+
+namespace godot {
+    class FmodEditorExportPlugin : public EditorExportPlugin {
+        GDCLASS(FmodEditorExportPlugin, EditorExportPlugin)
+
+    public:
+        void _export_begin(const PackedStringArray &features, bool is_debug, const String &path, uint32_t flags) override;
+        String _get_name() const override;
+
+        static void _bind_methods();
+
+        FmodEditorExportPlugin() = default;
+        ~FmodEditorExportPlugin() = default;
+
+    private:
+        PackedStringArray _list_files_in_folder(const String& folder, const String& extension = "");
```

Comment:
Wouldn't that be better to pass a PackedStringArray reference as a function parameter instead. We would just keep writing in the same variable, instead of creating a new one for each recursion level and appending it to the upper one afterward.